### PR TITLE
Release v1.4.557

### PR DESCRIPTION
**Pull request by mfisher87**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mfisher87** [comment]
@conda-forge-admin, please rerender

**Comment by mfisher87** [comment]
@conda-forge-admin, please rerender

File: recipe/patches/0001-always-build-quarto-preview-js.patch [hunk 1]
```diff
@@ -7,14 +7,15 @@ to remove this patch. See:
...
 +++ b/src/webui/quarto-preview/build.ts
-@@ -11,25 +11,7 @@ try {
+@@ -12,26 +12,7 @@ try {
+   jsBuildTime = 0;
  }
  
- // check if any of our repo files have a later time
--let build = false;
+-// check if any of our repo files have a later time
+-let build = args.length > 0 && args.includes("--force");
```
Comment (mfisher87) [inline recipe/patches/0001-always-build-quarto-preview-js.patch:18]: I feel I should somehow be leveraging this mechanism to force the build, but didn't have time to figure out whether / how I can influence that from our build script.

Comment (mfisher87) [reply recipe/patches/0001-always-build-quarto-preview-js.patch:18]: >So as you are calling prepare-dist already it should build. But the recipe is also calling configure first, so it builds a first time too
>
> ...
>
> Maybe this patch is not needed anymore ?

Thanks so much for helping with this context, Christophe! It's often easy to lose track of these things between releases!

I have a long holiday weekend for US independence day and I'll give this a test.

**Comment by mfisher87** [comment]
I don't understand why the checks aren't running. I'm going to wait I guess and see if an outage is happening. I tested the build locally without the patch and it works great :) Thanks @cderv!

**Comment by mfisher87** [comment]
@conda-forge-admin, please rerender

**Comment by mfisher87** [comment]
Yeah it's a "major outage" :) 

https://conda-forge.org/status/

**Comment by mfisher87** [comment]
@conda-forge-admin, please lint

**Comment by mfisher87** [comment]
This linting error is fixed in a recent PR (https://github.com/conda-forge/conda-smithy/pull/1969) but I think it needs to be released.

**Comment by cderv** [comment]
@mfisher87 if I can help with this, please let me know

**Comment by cderv** [comment]
Great ! 

Let's do latest stable 1.5 now, should we ?

**Comment by mfisher87** [comment]
Whew, wasn't aware it was ready :) Congrats!

I'm happy to provide any support I can during the week but won't have a solid block to work on this myself until the weekend.

**Comment by cderv** [comment]
Yeah 1.5 is out since 1st July and we patched it once last week on 8th. So we have new version to provide for conda user. 
No hurry, you're help is really appreciated on this, and this is cool when you can !

**Comment by mfisher87** [comment]
Are there any major changes to build I should be aware of? Or should I expect it to build similarly to 1.4?

**Comment by cscheid** [comment]
We have dependency bumps, but (intentionally!) no big changes to the build process itself since 1.4.

**Pull request merged**